src/consts.ts

```diff
@@ -6,15 +6,15 @@ import { stringToU8a } from '@polkadot/util';
 /*
  * Global Constants
  */
-export const URI_PREFIX = '/dashboard';
-export const TITLE_DEFAULT = 'Polkadot Staking Dashboard';
-export const DAPP_NAME = 'polkadot_staking_dashboard';
-export const POLKADOT_URL = 'https://polkadot.network';
+export const URI_PREFIX = '';
+export const TITLE_DEFAULT = 'Cere Staking Dashboard';
+export const DAPP_NAME = 'cere_staking_dashboard';
+export const CERE_URL = 'https://cere.network/';
 
 export const POLKADOT_ENDPOINT = 'wss://rpc.polkadot.io';
 export const WESTEND_ENDPOINT = 'wss://westend-rpc.polkadot.io';
-export const DEFAULT_NETWORK = 'polkadot';
-export const ACTIVE_NETWORK = 'polkadot';
+export const DEFAULT_NETWORK = 'cereMainnet';
```

Comment:
Why not just `cere`?

Reply:
Because we have `cereMainnet` and `cereTestnet`

Reply:
I see. It is a variable name.